### DISCUSSION TITLE
VSCode extension issues

### DISCUSSION THREAD

**Discussion by ionutpredoaia** [eclipse-sirius/sirius-components#3471] [Q&A]
Hello,

Is the VSCode extension from this repository actively maintained? 

I ran the VSCode extension and connected it to a localhost instance of Sirius Web, however the following aren't working properly:

**1)** when clicking on a model element from the tree editor (Explorer), I can view all of its properties in the Properties View (the form from the right side shown below); however, when I click on one of the fields from the Properties View, an error occurs as shown in the image below.

![extension_propertieserror](https://github.com/eclipse-sirius/sirius-web/assets/7106155/a7c16a79-1f46-406f-a103-3b21fa9660fe)

Note that this error only appears when using a localhost instance of Sirius Web; the error does not occur when connecting to https://siriusweb.obeostudio.com; What could be the reason for this?

**2)** when clicking on a representation, the diagram is not loaded and an error related to a "Validation error" occurs, as shown in the following image.

![extension_diagramerror](https://github.com/eclipse-sirius/sirius-web/assets/7106155/84e221c0-4524-4619-a0dd-78c867b80599)

This error occurs both with the localhost instance and with https://siriusweb.obeostudio.com.

It seems that the VSCode extension is out of sync with the latest version of Sirius Web.

Thanks,
Ionut

**Comment by sbegaudeau** [comment]
Hi,

The VS Code extension is being maintained, we will investigate this potential issue to fix it.

The instance of Sirius Web running on the URL that you are mentioning is a private test instance for the development team. This URL is not public and I don't know how you got access to it. The content of this instance can be altered and will be removed without notifications. Please do not use or share this URL.

The VS Code extension can be unsynchronized with the content of this instance but it should stay synchronized with the code in the repository. Feel free to open an issue if this bug is still present on your side.

Regards,

Stephane Begaudeau
